web/src/engine/main/src/keymanEngine.ts

```diff
@@ -84,7 +84,8 @@ export default class KeymanEngine<
     }
 
     return {
-      keyboardInterface: this.interface,
+      keyboardInterface: this.interface ||
+        new KeyboardInterface<ContextManager>(window, this, this.config.stubNamespacer),
```

Comment:
I don't understand this -- wouldn't it be cleaner to instantiate `this.interface` once in the constructor and then just reference it here?

```suggestion
      keyboardInterface: this.interface,
```